schedule/__init__.py

```diff
@@ -341,16 +341,29 @@ def at(self, time_str):
         :param time_str: A string in `XX:YY` format.
```

Comment:
The docstring needs an update because we now also support different time formats here. I'm thinking the explainer text also needs a once-over because we substantially increased the abilities of this function.

Reply:
I made some changes, let me know what you think.  I tried to keep it short but accurate to follow your existing comment style, which I perceived as prioritizing conciseness.  Let me know if I should have written more.